src/MoBi.Presentation/Presenter/SelectReferencePresenterBase.cs

```diff
@@ -153,17 +153,15 @@ public ReferenceDTO GetReferenceObjectFrom(ObjectBaseDTO objectBaseDTO)
             if (string.Equals(objectBaseDTO.Id, AppConstants.Time))
                return timeReference();
 
-            var objectBase = _context.Get<IObjectBase>(objectBaseDTO.Id);
-            if (objectBase != null)
-               return createPathsFromEntity(objectBase);
-
```

Comment:
This old way relied on the fact that we built certain DTO's with dummy Id's. If you construct the dto using the underlying object Id, then this order breaks the path creation algorithm.